lib/active_merchant/billing/gateways/flex_charge.rb

```diff
@@ -115,23 +135,29 @@ def add_invoice(post, money, credit_card, options)
           avsResultCode: options[:avs_result_code],
           cvvResultCode: options[:cvv_result_code],
           cavvResultCode: options[:cavv_result_code],
-          cardNotPresent: credit_card.verification_value.blank?
+          cardNotPresent: credit_card.is_a?(String) ? false : credit_card.verification_value.blank?
         }.compact
       end
 
       def add_payment_method(post, credit_card, address, options)
```

Comment:
The credit_card name could be something more generic, like payment_method considering this is not always a credit_card

Reply:
I saw that, but @Heavyblade is still working on flex charge so changing this argument led to merging problems that we are not willing to deal with at this moment.